plugins/usertiming.js

```diff
@@ -0,0 +1,93 @@
+/**
+\file usertiming.js
+Plugin to collect metrics from the W3C User Timing API.
+For more information about User Timing,
+see: http://www.w3.org/TR/user-timing/
+*/
+
+(function() {
+
+	BOOMR = BOOMR || {};
+	BOOMR.plugins = BOOMR.plugins || {};
+
+	/**
+	 * @returns String compressed user timing data that occurred since the last call
+	 */
+	function getUserTiming() {
+		var now = BOOMR.now();
+
+		map = UserTimingCompression.getCompressedUserTiming(impl.options);
+		res = UserTimingCompression.compressForUri(map);
+		impl.options["from"] = now;
```

Comment:
it is be better to use `options["to"]` since it's possible for future `UserTiming` entries to have a start time before `now`, but an end time after `now`.  We want to capture all `UserTiming` entries that became available since the last beacon, so I would say we should capture everything that ended after the last beacon regardless of when it started.